### ISSUE TITLE
Location input field are output with a long label.

### ISSUE THREAD

**Issue by Ram-G** [Azure/LogicAppsUX#3553]
### Describe the Bug with repro steps

1. Open a flow which contains actions has long placeholder text
2. In Power Automate, one example is the Excel 'delete a row' action.
3. Observe placeholder label.

**Actual Result:**
Location input fields are output by the screen reader with a long label, such as the "Select from the drop-down or specify one of the following: - "me" - "SharePoint Site URL" - "users/someone's UPN" - "groups/group Id" - "sites/SharePoint Site URL:/teams/team name:" (the colons are required)" This means that blind users have to wait for a long output to be able to perceive all the information.

**Expected Result:**
Additional information should be short and meaningful. If short wording is not possible, it should be moved out to a help icon. This icon should:
be placed in front of the input field,
receive the focus with the tab key,
display the hint when focus is received, and
be output by the screen reader.

### What type of Logic App Is this happening in?

Consumption (Portal)

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Screenshots or Videos

![image](https://github.com/Azure/LogicAppsUX/assets/10837129/74d62df9-ad64-4edb-bd3b-e62ab3ed9868)

### Browser

Edge

### Additional context

Related internal bug: https://msazure.visualstudio.com/One/_workitems/edit/13921602

AB#25622474

**Comment by hartra344** [comment]
This is because the description field being sent for the Location Field is entirely too long, this should be fixed in the manifest, not change how the UI works because of it.

I also agree that we don't necessarily need to have screen readers read the placeholder value of the text box but this seems to be fluentUI's behavior, not ours.
